docs/build-for-developers/cli-usage.md

```diff
@@ -171,16 +214,16 @@ running workflows via the CLI.
 
 ### Load adaptor documentation
 
-The CLI can list adaptor documentation in the terminal. Note that it has to download
-the adaptor to the repo (if it's not already there), which can take a moment.
+The CLI can list adaptor documentation in the terminal. Note that it has to
+download the adaptor to the repo (if it's not already there), which can take a
+moment.
```

Comment:
Just a question, After downloading the adaptor documentation 
Next time you run the `openfn docs http` for example. It will fetch from repo right ?

Reply:
From the local repo, yes. It will cache the documentation.